### PR TITLE
Feat/k8s node pool public ips

### PR DESCRIPTION
**Pull request by mflorin**
Docs updated

### CODE REVIEW

**Comment by mflorin** [comment]
@piepmatz - I've implemented your comments in this pull request. Thanks.

File: k8s.go [hunk 1]
```diff
@@ -261,7 +261,7 @@ type KubernetesNodePoolProperties struct {
 
 	// Public Ips
 	// Required: false
-	PublicIps	[]string	`json:"publicIps,omitempty"`
+	PublicIPs *[]string `json:"publicIps,omitempty"`
```
Comment (piepmatz) [inline k8s.go:264]: Why do you need a pointer here even though you can also assign `nil` to `[]string`?

Comment (mflorin) [reply k8s.go:264]: Agreed, however, I was trying to follow the general approach for the rest of the structure (see LANs) - thought it was preferred that way. Will revert it, no problem.

Comment (mflorin) [reply k8s.go]: @piepmatz - done

Comment (piepmatz) [reply k8s.go]: Alright, thx. I do value consistency a lot, but in this case I'd rather go for the better solution than sticking to the pointer approach taken for LANs.

**Pull request merged**